api/scpca_portal/models/project.py

```diff
@@ -484,6 +513,8 @@ def update_project_aggregate_properties(self):
             project_summary.sample_count = count
             project_summary.save(update_fields=("sample_count",))
 
+        return self.summaries.all()
```

Comment:
```suggestion
```
Was this just added for testing? If we don't use it we should remove.

Reply:
Yes, it was added for testing - now removed 👍